gitpod-io/gitpod
# [dashboard] Admin dashboard

Adds the admin dashboard to the new dashboard.

How to test:
 - create a user with 'admin' flag (default in dev staging)
 - you should see the admin menu on top.
 - try search for users or workspaces.
